### PR TITLE
Quiet Flag For CLI

### LINKED ISSUE #105: output nothing to stdout
if this [PR](https://github.com/Boeing/config-file-validator/pull/104) is merged, i want `--quiet` option which outputs nothing to stdout.

### PR DESCRIPTION
**Pull request by HakanVardarr**
tried to implement quiet flag. Added a global variable Quiet and WithQuiet function in cli.go. I forgot to remove the env implementation. I took the easy way with the global variable if you don't like it I can try to find another way.

Closed #105

### CODE REVIEW

**Comment by kehoecj** [comment]
Thanks for another PR @HakanVardarr ! Can you take out the env changes so this is a standalone PR?

**Comment by HakanVardarr** [comment]
@kehoecj I am new to pull request stuff sorry for the mistake. I am trying to learn. :)

**Review by kehoecj — COMMENTED** [review]
Found the following issue during functional testing. Running `>> docker run -v $(pwd)/test:/test cfv:quiet --depth 0 /test` outputs the following to stdout:

```
Summary: 0 succeeded, 0 failed
```

File: cmd/validator/validator.go [hunk 1]
```diff
@@ -78,6 +79,7 @@ func getFlags() (validatorConfig, error) {
 	reportTypePtr := flag.String("reporter", "standard", "Format of the printed report. Options are standard and json")
 	versionPtr := flag.Bool("version", false, "Version prints the release version of validator")
 	groupOutputPtr := flag.String("groupby", "", "Group output by filetype, directory, pass-fail. Supported for Standard and JSON reports")
+	quietPrt := flag.Bool("quiet", false, "If quiet flag is set. It doesn't prints anything to stdout.")
```
Comment (kehoecj) [inline cmd/validator/validator.go:82]:
```suggestion
	quietPrt := flag.Bool("quiet", false, "If quiet flag is set. It doesn't print any output to stdout.")
```

**Comment by HakanVardarr** [comment]
@kehoecj I'll look for thath issue

**Comment by HakanVardarr** [comment]
You didn't pass the quiet flag in the test

```bash
 docker run -v $(pwd)/test:/test cfv:quiet --depth 0 /test
 ```
 
 ```bash
 docker run -v $(pwd)/test:/test config-file-validator:v1.5.0 --quiet --depth 0 /test
 ```
 
 I ran this and I didn't get any output.
Screenshot 2024-03-29 at 21 17 42

**Comment by kehoecj** [comment]
> You didn't pass the quiet flag in the test
> 
> ```shell
>  docker run -v $(pwd)/test:/test cfv:quiet --depth 0 /test
> ```
> 
> ```shell
> docker run -v $(pwd)/test:/test config-file-validator:v1.5.0 --quiet --depth 0 /test
> ```
> 
> I ran this and I didn't get any output. Screenshot 2024-03-29 at 21 17 42

You're right! The container name threw me off. I re-ran with `--quiet` and it works as expected. Thanks for QA-ing my QA 😆

**Review by kehoecj — APPROVED** [review]
LGTM - great work

**Comment by HakanVardarr** [comment]
> > You didn't pass the quiet flag in the test
> > ```shell
> >  docker run -v $(pwd)/test:/test cfv:quiet --depth 0 /test
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ```shell
> > docker run -v $(pwd)/test:/test config-file-validator:v1.5.0 --quiet --depth 0 /test
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > I ran this and I didn't get any output. Screenshot 2024-03-29 at 21 17 42
> 
> You're right! The container name threw me off. I re-ran with `--quiet` and it works as expected. Thanks for QA-ing my QA 😆

If it is working that is good. 😄

**Comment by ccoVeille** [comment]
Thanks a lot for implementing the quiet mode

**Pull request merged**